specs/capella/beacon-chain.md

```diff
@@ -336,18 +338,38 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
     process_sync_aggregate(state, block.body.sync_aggregate)
 ```
 
-#### New `get_expected_withdrawals`
+#### New `get_balance_minus_withdrawals`
 
 ```python
-def get_expected_withdrawals(state: BeaconState) -> Sequence[Withdrawal]:
-    epoch = get_current_epoch(state)
-    withdrawal_index = state.next_withdrawal_index
-    validator_index = state.next_withdrawal_validator_index
+def get_balance_minus_withdrawals(
+    state: BeaconState,
+    validator_index: ValidatorIndex,
+    withdrawals: Sequence[Withdrawal],
+) -> Gwei:
+    total_withdrawn = sum(w.amount for w in withdrawals if w.validator_index == validator_index)
```

Comment:
a little pedantic: id be tempted to call it something like `validator_balance_withdrawn` or even just `withdrawn` rather than `total_withdrawn` just because its the balance withdrawn for this specific validator and not the `total_withdrawn`